sources/packages/web/src/services/UserService.ts

```diff
@@ -37,4 +38,17 @@ export class UserService {
   async checkActiveUser(authHeader?: any): Promise<boolean> {
     return ApiClient.User.checkActiveUser(authHeader);
   }
+
+  async getInstitutionUser(
+    authHeader?: any,
+  ): Promise<InstitutionUserDetailsDto> {
+    return ApiClient.User.getinstitutionUser(authHeader);
+  }
+
+  async updateInstitutionUser(
+    data: InstitutionUserDetailsDto,
+    authHeader?: any,
+  ): Promise<void> {
+    return ApiClient.User.updateInstitutionUser(data, authHeader);
```

Comment:
promising void and you are returning a value

Reply:
return statement has been removed.